build/documentation/generate-typedocs.js

```diff
@@ -40,11 +44,19 @@ process.argv.forEach(function (val, index) {
     var validArg = true;
 
     if (index === 2) {
-        val === "--dry-run" ? dryRun = true : validArg = false;
+        if(val === "--dry-run") {
```

Comment:
It might be better to use a switch statement for performance and also I could see where someone may want to use both arguments for dry run and verbose at the same time.